src/plugins/home/public/application/components/guided_onboarding/getting_started.scss

```diff
@@ -0,0 +1,3 @@
+.gettingStarted__panel {
```

Comment:
We should avoid creating new SCSS files in favor of Emotion, except in rare circumstances.